unlock-app/src/queries/keysCount.ts

```diff
@@ -4,7 +4,7 @@ export default function keysCount() {
   return gql`
     query keysCount($addresses: [String!], $timestamp: BigInt! = 0) {
       totalKeys: locks(where: { address_in: $addresses }) {
-        keys(where: { expiration_gt: 0 }) {
+        keys(where: { expiration_gt: 0 }, first: 1000) {
```

Comment:
Set max value for results entries

Reply:
Is that going to break components who are using this query?

Reply:
Query used to retrieve active/total members, used only in this section.
Now the total members is fetched from the contract, and the active members is get from graphQL, value is set to 1000 (max) to prevent the same issue for the total (need to replace if later and get active members from contract)